protocol/protocol_state.re

```diff
@@ -1,16 +1,17 @@
 open Helpers;
 open Crypto;
 
-module Operation_side_chain_set = Set.Make_with_yojson(Operation.Side_chain);
-module Operation_main_chain_set = Set.Make_with_yojson(Operation.Main_chain);
+module Tezos_operation_set =
+  Set.Make_with_yojson(Protocol_operation.Core_tezos);
+module User_operation_set =
+  Set.Make_with_yojson(Protocol_operation.Core_user);
 
 type t = {
-  // state machine data
-  ledger: Ledger.t,
+  core_state: Core.State.t,
   // TODO: more efficient lookup on included_operations
   // TODO: is this part of the protocol?
-  included_operations: Operation_side_chain_set.t,
-  included_main_operations: Operation_main_chain_set.t,
+  included_tezos_operations: Tezos_operation_set.t,
```

Comment:
Why reorder fields?

Reply:
As I was touching the lines I reorder it to keep consistent with everywhere else. First we handle Tezos_operations then User_operations.